utils/js-wrapper/examples/guestbook/wrapper/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use codec::{Decode, Encode};
+use gstd::prelude::*;
+use scale_info::TypeInfo;
+use serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+pub fn decode<D: Decode>(bytes: Vec<u8>) -> Result<D, String> {
+    D::decode(&mut bytes.as_ref()).map_err(|_| String::from("Unable to decode"))
+}
+
+#[derive(TypeInfo, Decode, Encode, Serialize)]
+pub struct MessageIn {
+    pub author: String,
+    pub msg: String,
+}
+
+#[wasm_bindgen]
+pub fn __send_add_message(author: String, msg: String) -> Vec<u8> {
+    let mut v: Vec<u8> = Vec::new();
+    v.push(0);
```

Comment:
This kind of pushing is discussable I think. I don't find that wrapper is enough comfortable to use and write manually